pyfolio/tears.py

```diff
@@ -58,12 +58,12 @@ def create_full_tear_sheet(returns,
                            transactions=None,
                            market_data=None,
                            benchmark_rets=None,
-                           gross_lev=None,
                            slippage=None,
                            live_start_date=None,
                            sector_mappings=None,
                            bayesian=False,
                            round_trips=False,
+                           estimate_intraday=None,
```

Comment:
I see, `None` means to infer. This appears a bit brittle. Instead I would do `estimate_intraday='infer'` to make this explicit.